### ISSUE TITLE
status of this project?

### ISSUE THREAD

**Issue by vitamin5x** [menacher/java-game-server#76]
Is this project alive? anybody on this project?
It seems that there is not any update since two years age

**Comment by t3573393** [comment]
may be is down..

**Comment by menacher** [comment]
Active feature development is not ongoing. However, developers do provide pull requests and maintenance tasks can be done on request.

**Comment by datotahost** [comment]
yes ,my team working on this project with unity,spring framework for real money poker game
